browser/css/jssidebar.css

```diff
@@ -143,6 +153,11 @@
 	justify-content: space-between;
 }
 
+#textorientbox.sidebar button.ui-pushbutton {
```

Comment:
I have proposal to add these "custom" rules for specific objects at the end of file in one section like:
/* CUSTOMIZED WIDGETS */
so it will be easy to find non-generic rules.